metrics.nim

```diff
@@ -4,6 +4,17 @@
 #   * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or http://www.apache.org/licenses/LICENSE-2.0)
 # at your option. This file may not be copied, modified, or distributed except according to those terms.
 
+# This is not quite true for several portions of the metrics library, mainly
+# because async is broken and raises lots of naked Exception that can't be
+# tracked - nonetheless we'll make an effort to catch at least the exceptions
+# that can be tracked - nobody knows what can happen in the rest of the code,
+# so don't use metrics if you value stability.
```

Comment:
Seriously?

Reply:
well, they're untracked and can change when the std lib changes - for example there are untracked errors coming out of the `async` macro that can't be try/caught

Reply:
ok, reworded to something more informative